collada/__init__.py

```diff
@@ -219,8 +219,11 @@ def __init__(self,
             xml_root = self.xmlnode.getroot()
             if hasattr(xml_root, 'nsmap'):
                 # lxml has an nsmap
-                # use the first value in the namespace map
-                namespace = next(iter(xml_root.nsmap.values()))
+                # use the latest collada xsd in the namespace map
+                namespaces = xml_root.nsmap.values()
+                collada_namespaces = filter(lambda uri: 'COLLADA' in uri.upper(), namespaces)
```

Comment:
what if this filters down to nothing? would it be better to fall back to using the first one if none match "collada" string?